### PR TITLE
Add Deletion of Alternate nodes in a linked list C++

### LINKED ISSUE #572: Deletion of Alternate nodes in a linked list
[C++] This program will delete alternate nodes in the linked list.

For example : The linked list provided by the user is: 1->2->3->4->5->6->7->8->9
Output would be :  1->3->5->7->9

Please assign me this issue.

### PR DESCRIPTION
**Pull request by Mohim-Singla**
fixed #572 
Adding program for deletion of alternate nodes of a linked list using c++

### CODE REVIEW

File: Code/C++/Deletion of Alternate nodes in a linked list.cpp [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+After creating a linked list, call the alternate_delete() function.
+Traverse the list and unlink/delete and free data of every node at even position.
+*/
+
+
+#include <iostream>
+#include <stdlib.h>
+using namespace std;
+
+struct node{
```
Comment (rudrakshi99) [inline Code/C++/Deletion of Alternate nodes in a linked list.cpp:15]: use `class` instead of `struct`.

**Review by rudrakshi99 — CHANGES_REQUESTED** [review]

**Review by rudrakshi99 — COMMENTED** [review]
use `nullptr` instead of `NULL

**Comment by Mohim-Singla** [comment]
@rudrakshi99  I have made the changes. Please check.

**Review by rudrakshi99 — APPROVED** [review]

**Review by nakul-19 — CHANGES_REQUESTED** [review]
@Mohim-Singla please make changes in the respective readme.

**Comment by Mohim-Singla** [comment]
@nakul-19  Updated readme.md and resolved conflicts

**Pull request merged**